server/cluster/metrics.go

```diff
@@ -119,6 +119,14 @@ var (
 			Name:      "store_sync",
 			Help:      "The state of store sync config",
 		}, []string{"address", "state"})
+
+	regionStateGauge = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
```

Comment:
A bit of a duplication with `regionStatusGauge` of `server/statistics/metrics.go`.